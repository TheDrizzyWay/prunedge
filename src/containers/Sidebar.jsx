import React from 'react';
import User from '../components/User';
import Menu from '../components/Menu';

const Sidebar = () => {
    return (
        <div className="sidebar">
            <User />
            <Menu />
        </div>
    );
}

export default Sidebar;
