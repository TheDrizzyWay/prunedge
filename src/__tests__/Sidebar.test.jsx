import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Sidebar from '../containers/Sidebar';

configure({ adapter: new Adapter() });

describe('Sidebar component', () => {
  it('should render sidebar', () => {
    const wrapper = shallow(
      <Sidebar />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
