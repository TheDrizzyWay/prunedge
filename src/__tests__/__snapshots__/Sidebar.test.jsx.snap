// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar component should render Content 1`] = `ShallowWrapper {}`;

exports[`Sidebar component should render sidebar 1`] = `ShallowWrapper {}`;
